mnemonic-core/src/test/java/org/apache/mnemonic/Person.java

```diff
@@ -71,6 +75,13 @@ public abstract void setName(String name, boolean destroy)
       throws OutOfHybridMemory, RetrieveDurableEntityError;
 
   @DurableGetter
+  public abstract MemBufferHolder getNamebuffer();
```

Comment:
Could you figure out a meaningful name for the buffer usage ? e.g. figureprint, picture, signature ?